packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -177,15 +189,21 @@ const ClinicalResultBloodPressure = (props) => {
 
   const clinicalResultBloodPressureDisplay = (
     <React.Fragment>
-      <div className={decoratedResultClassnames}>
-        <div className={cx('result-display')}>
-          {decoratedResultDisplay}
-          {isTruncated ? null : !hideAccessoryDisplays && iconGroupDisplayElement}
+      <span
+        role={IsBloodPressureAGroup ? 'group' : undefined}
+        aria-labelledby={IsBloodPressureAGroup ? `id-hidden-text ${idForDatetimeDisplays}` : undefined}
+      >
+        <div className={decoratedResultClassnames}>
+          <div className={cx('result-display')}>
+            {IsBloodPressureAGroup && <VisuallyHiddenText id="id-hidden-text" aria-hidden="true" text="Blood Pressure" />}
```

Comment:
I need to update the hardcoded string to be from the translation file.

Reply:
For this line if aria-hidden is true for the VisuallyHiddenText then we won't see it on the front end and the screenreader will skip it - so it wouldn't be seen or heard. Is that the expected behavior for this?

Reply:
Yup, we need this to be read only when the prop is set to true. If the prop is false, then just the result needs to be read out. 
@brooksienoodlesoup - Can you confirm how the behavior of this should be?

Reply:
Updated to use translation string - https://github.com/cerner/terra-clinical/pull/868/commits/ba6cb16362db09a115cd6bb88d36f7853bfabc10#diff-42afe5315e2b63b43f755f1ec567ef32b1ad903109d718db4e7526f825680f6aR197. 

Left a comment on Reagan's [Jira](https://jira.cerner.com/browse/TRANS-14531?focusedCommentId=4752432&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4752432) asking if they need a new Jira or if they can translate to non-english ones as part of the same Jira.

Reply:
This will be done as part of Reagan's story - https://jira.cerner.com/browse/TRANS-14531?focusedCommentId=4752846&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4752846

Reply:
I think I'm still confused, here even when isBloodPressureGrouped true the text won't be seen or read out

Reply:
The text 'Blood Pressure' is never going to be seen irrespective of whether isBloodPressureGrouped is true or false. If isBloodPressureGrouped is true, then the screen reader should read it.

Reply:
Yup that first part makes sense about it not being seen, but the screenreader won't read it out at all if aria-hidden is set to true. Right? That was my understanding: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden